controller/src/main/java/io/pravega/controller/server/rest/generated/model/CreateStreamRequest.java

```diff
@@ -40,6 +40,9 @@
   @JsonProperty("streamTags")
   private TagsList streamTags = null;
 
+  @JsonProperty("rolloverSizeBytes")
```

Comment:
NOTE: This class is auto generated by the swagger code generator program

Reply:
Thanks for pointing out. Yes this is generated code, I follow the guide here to generate it https://github.com/pravega/pravega/tree/master/shared/controller-api/src/main/swagger